src/features/advanced-content-insights/AdvancedContentInsightsToggle.tsx

```diff
@@ -11,19 +11,26 @@ import messages from './messages';
 import './AdvancedContentInsightsToggle.scss';
 
 interface Props {
+    hasDescription?: boolean;
```

Comment:
are the description and tooltip now independent? does that mean someone could use both at the same time? is that intended?

Reply:
yep, we want to have it controlled by separate prop ,but we can add !hasTooltip flag to avoid having both tooltip and description at the same time

Reply:
is there a scenario in which we show neither a tooltip nor description? if not, why do we need the separate prop?

Reply:
thanks, good question. previously description was based on toggle state (!isChecked), but product requirement is different now and we need to add more conditions for description to show up. that why we need separate prop

Reply:
couldn't they still be controlled with one prop? the additional conditions would just be added into the `hasTooltip` logic

for example:
```
// instead of:
<AdvancedContentInsightsToggle
    hasDescription={condition && otherCondition}
    hasTooltip={true}
    ...
/>

// it would be:
<AdvancedContentInsightsToggle
    hasTooltip={!(condition && otherCondition)}
    ...
/>
```

this is with the assumption that if the description conditions is true, then the tooltip is not shown. but if the description is not shown, then the tooltip is rendered